Sources/AuthFoundation/Requests/Token+Requests.swift

```diff
@@ -82,10 +82,12 @@ extension Token.IntrospectRequest: OAuth2APIRequest, APIRequestBody {
     var httpMethod: APIRequestMethod { .post }
     var contentType: APIContentType? { .formEncoded }
     var acceptsType: APIContentType? { .json }
-    var authorization: APIAuthorization? { token }
+    var authorization: APIAuthorization? { nil }
+    private var clientId: String? { token.idToken?[.clientId] }
     var bodyParameters: [String: Any]? {
         [
             "token": (token.token(of: type) ?? "") as String,
+            "client_id": token.context.configuration.clientId,
```

Comment:
Can you add a test that expects this param to exist? 

Here's one I did in kotlin: https://github.com/okta/okta-mobile-kotlin/blob/master/auth-foundation/src/test/java/com/okta/authfoundation/client/OidcClientTest.kt#L211

Reply:
Good call out @JayNewstrom . I have include test for that case